src/common/string.cpp

```diff
@@ -2302,4 +2302,3 @@ int wxString::Freq(wxUniChar ch) const
     }
     return count;
 }
-
```

Comment:
Please revert the accidental whitespace change, which is the only change in the file.